frontend/packages/client/src/components/Community/CommunityCard.js

```diff
@@ -29,9 +30,31 @@ const CommunityCard = ({
 
   const hoverClasses = !isComingSoon ? " transition-all community-card" : "";
 
+  const { width: windowWidth } = useWindowDimensions();
+  const joinBtnPositions = useMemo(() => ({
+    bigger: { top: "52px", right: "74px" },
+    smaller: { top: "144px", right: "332px" }
+  }), []);
+  const joinBtnDefault = windowWidth >= 500 ? joinBtnPositions.bigger : joinBtnPositions.smaller;
+  const [joinBtnTopRight, setJoinBtnTopRight] = useState(joinBtnDefault);
+  const [bottomSpacer, setBottomSpacer] = useState();
+
+  useEffect(() => {
+    if (windowWidth < 500 && joinBtnTopRight.top !== joinBtnPositions.smaller.top && bottomSpacer === false) {
+      setJoinBtnTopRight({
+        top: joinBtnPositions.smaller.top,
+        right: joinBtnPositions.smaller.top - windowWidth + 500
+      });
+      setBottomSpacer(true);
+    } else if (windowWidth >= 500 && joinBtnTopRight.top !== joinBtnPositions.bigger.top && bottomSpacer) {
+      setJoinBtnTopRight(joinBtnPositions.bigger);
+      setBottomSpacer(false);
+    }
+  }, [joinBtnTopRight.top, windowWidth, bottomSpacer, joinBtnPositions]);
+
```

Comment:
is it possible to achieve all of this positioning with flex containers?  on mobile columns and column classes, in addition using custom components for mobile or desktop could help

Reply:
I've created this ticket to track the improvement https://linear.app/dappercollectives/issue/CAS-103/improve-render-for-join-community-button-on-communitycard